projects/storefrontlib/src/cms-components/checkout/components/po-number/po-number.component.spec.ts

```diff
@@ -22,6 +22,7 @@ import createSpy = jasmine.createSpy;
 class MockSpinnerComponent {}
 
 class MockPaymentTypeService {
+  readonly ACCOUNT_PAYMENT = 'ACCOUNT';
   getSelectedPaymentType(): Observable<string> {
     return of('ACCOUNT');
```

Comment:
I guess we could use the new `ACCOUNT_PAYMENT` constant as a value here.

Reply:
done